journal_rec_word_vectors.py

```diff
@@ -0,0 +1,42 @@
+from io import BytesIO
+from pathlib import Path
+
+from gensim.models import Word2Vec
+from gensim.parsing.preprocessing import remove_stopwords
+import numpy as np
+
+from pdfminer.high_level import extract_text
+
+word_model = Word2Vec.load("data/word2vec_model/biorxiv_300.model")
```

Comment:
The argument is a relative path. Is it going to cause any trouble when this module is imported from other places?

Reply:
This path is for local testing and is relative to my repository. Not a problem for now as this repository is designed to be deployed as a website. It may get a bit tricky when I get ready for full deployment as github's file size restrictions can be debilitating.